andela/resly
# Ft restaurant settings 117808237

#### What does this PR do?

This PR sets the refund for a cancelled reservation depending on the refund rate of the restaurant.
#### ​Description of Task to be completed?

The owner of a restaurant can set the rate of refund for his restaurant.
When a diner cancels a reservation made on a restaurant, he gets a refund.
The refund is based on the refund rate set by the owner of the restaurant.
#### How should this be manually tested?

> Visit the landing page and login.
> Make a reservation and complete the payment.
> Go to `Your Profile` > `Current Reservations`
> Click on `Cancel` to confirm and cancel the reservation.
> Visit `Cancelled Reservations`
> The refund is applied to that cancelled reservation.
#### ​Any background context you want to provide?

It is frequent for diners to cancel their reservations. The rate of refund should not just be static but dynamic.
#### ​What are the relevant pivotal tracker stories?

The story relating to this PR can be view [here](https://www.pivotaltracker.com/projects/1415814/stories/117808237)
#### ​Screenshots​

![screen shot 2016-05-09 at 12 28 30 pm](https://cloud.githubusercontent.com/assets/17028608/15112176/5a1b963a-15e4-11e6-8ca0-57ccae64ca68.png)
![screen shot 2016-05-09 at 12 33 40 pm](https://cloud.githubusercontent.com/assets/17028608/15112179/5a8b4584-15e4-11e6-8649-24fca57bc567.png)
![screen shot 2016-05-09 at 12 34 18 pm](https://cloud.githubusercontent.com/assets/17028608/15112177/5a1c4f08-15e4-11e6-837b-fff17c1bdbaa.png)
![screen shot 2016-05-09 at 12 34 39 pm](https://cloud.githubusercontent.com/assets/17028608/15112178/5a1e1d60-15e4-11e6-9113-6935353e57c1.png)
#### ​Questions

There are no questions at this moment.
